rs/cross-chain/proposal-cli/src/candid/tests.rs

```diff
@@ -29,7 +29,10 @@ fn should_encode_non_empty_ledger_upgrade_args() {
 
     let upgrade_args = encode_upgrade_args(&path, "(variant {Upgrade})");
 
-    assert_matches!(upgrade_args.upgrade_args_hex(), _string);
+    assert_matches!(
+        hex::encode(upgrade_args.upgrade_args_bin()).as_str(),
+        "4449444c1e6b02fcb88b840301b0ced18403186e026c0992cdb6f902039efeb9a40309f2c794ae030aefcee780040b8484d5c0070585f199f40710b0d7c3920b1591c9aafe0d09bea3d1c30f166e046c089ea581d20105b2a7c2d20305a495a5e90605ffb08aab0806e0ab86ef0805e4d8cce80b0593c8e6c70c05dec5d8ae0e086e786e076d686e686e716e7d6e0c6d0d6c020071010e6b04cf89df017cc189ee017dfdd2c9df020fcdf1cbbe03716d7b6e116b029d83f46a12c9c5f1d0037f6c02b3b0dac30368ad86ca8305136e146d7b6e7a6e176c01c7bfe7b60b7e6c0dc295a99301199efeb9a40371f2c794ae037defcee780040caecbeb880412b2a4dab2051a97aabdbb06058484d5c0070582babe82081ca1e5f7a10a1db0d7c3920b1591c9aafe0d71bea3d1c30f166e7b6d1b6c020012017d6e126c089ea581d20178b2a7c2d20305a495a5e90678ffb08aab0806e0ab86ef0805e4d8cce80b0593c8e6c70c05dec5d8ae0e6801000000"
+    );
```

Comment:
The previous version of this test seems to have accepted any string. Initially the test was similar to what is being proposed in this PR, but maybe it was deemed to brittle? I'd be happy to change it to something else, e.g., to verify that the returned value is non-empty and different from `"4449444c0000"`?

Reply:
> but maybe it was deemed to brittle?

Yes, the problem is that a backward-compatible change is the candid file will result in another encoding and so this test will break, which is not what we want. This is just a smoke test, so maybe just testing that the hex encoded string starts with the [DIDL](https://mmapped.blog/posts/20-candid-for-engineers#encoding-an-empty-tuple) prefix(`4449444c`) is good enough?

Reply:
I changed the test as suggested, PTAL!